blockchain/blockdao/blockdao.go

```diff
@@ -680,20 +681,24 @@ func (dao *blockDAO) deleteTipBlock() error {
 	if err := whichDB.Commit(batchForBlock); err != nil {
 		return err
 	}
+	// Delete hash -> height mapping
+	hashKey := append(hashPrefix, hash[:]...)
```

Comment:
put it into a private function hashKey(hash)

Reply:
fixed in lastest commit